d2l-icon.js

```diff
@@ -127,7 +127,8 @@ Polymer({
 				this._iconset = /** @type {?Polymer.Iconset} */ (
 					this._meta.byKey(this._iconsetName));
 				if (this._iconset) {
-					this._iconset.applyIcon(this, this._iconName, this.theme);
+					var iconName = 'd2l-icon-' + this._iconName;
```

Comment:
If someone were to use `d2l-icon` with an iconset other than one of ours, this would fail. Is there an easy way to check if `this._iconset` contains `d2l-icon-{name}` and if not check for `{name}`?

Reply:
We do this with our profile icon in BSI actually:
https://github.com/Brightspace/brightspace-integration/blob/master/web-components/navigation-icons.html#L6

Reply:
Nice!